asapdiscovery-ml/asapdiscovery/ml/schema_v2/config.py

```diff
@@ -0,0 +1,1252 @@
+from __future__ import annotations
+
+import abc
+import pickle as pkl
+from collections.abc import Iterator
+from enum import Enum
+from pathlib import Path
+from typing import Callable, ClassVar
+
+import mtenn
+import numpy as np
+import torch
+from asapdiscovery.data.schema import ExperimentalCompoundDataUpdate
+from asapdiscovery.data.schema_v2.complex import Complex
+from asapdiscovery.data.schema_v2.ligand import Ligand
+from asapdiscovery.ml.dataset import DockedDataset, GraphDataset, GroupedDockedDataset
+from asapdiscovery.ml.es import BestEarlyStopping, ConvergedEarlyStopping
+from pydantic import BaseModel, Field, root_validator
+
+
+class OptimizerType(str, Enum):
```

Comment:
You can use the new `asapdiscovery.data.enum.StrEnum` that unifies methods for this string mixin'd enum.